be/src/exec/pipeline/pipeline_driver_poller.cpp

```diff
@@ -117,8 +117,14 @@ void PipelineDriverPoller::run_internal() {
 
 void PipelineDriverPoller::add_blocked_driver(const DriverRawPtr driver) {
     std::unique_lock<std::mutex> lock(this->_mutex);
+    driver->_pending_watcher->stage_start();
     this->_blocked_drivers.push_back(driver);
     this->_cond.notify_one();
 }
 
+void PipelineDriverPoller::remove_blocked_driver(DriverList& local_blocked_drivers, DriverList::iterator& driver_it) {
+    (*driver_it)->_pending_watcher->stage_stop();
```

Comment:
_pending_timer_sw.stop()
_pending_timer->update(_pending_timer->elapsed_time());

_total_timer can be handled in the same way as _pending_timer

Reply:
fixed